kahuna/public/js/search/index.js

```diff
@@ -88,11 +92,25 @@ search.config(['$stateProvider', '$urlMatcherFactoryProvider',
                 description: 'Toggle metadata panel',
                 callback: panels.metadataPanel.toggleHidden
             });
+
+            keyboardShortcut.bindTo($scope).add({
+                combo: shortcutKeys.get('showCollections'),
+                description: 'Toggle collections',
+                callback: () => {
+                    const toggled = !storage.getJs('showCollectionsPanel');
+                    storage.setJs('showCollectionsPanel', toggled);
+                    $window.alert('Now refresh to start using collections');
+
+                }
+            });
         }],
         resolve: {
             shortcutKeys: [function() {
                 // keep the shortcut keys here to stop overriding
-                return new Map([['metadataPanel', 'm']]);
+                return new Map([
+                    ['metadataPanel', 'm'],
+                    ['showCollections', 'ctrl+c']
```

Comment:
Is there not a risk that this conflicts with the standard "Copy" shortcut (eg copying text)? Is the assumption that people are all on Mac and Ctrl+C != Cmd+C?